application-services/custom/cloud-event/go.mod

```diff
@@ -1,10 +1,20 @@
 module cloud-event
 
-go 1.15
+go 1.16
 
 require (
-	github.com/cloudevents/sdk-go v1.1.2
-	github.com/edgexfoundry/app-functions-sdk-go v1.3.0
-	github.com/edgexfoundry/go-mod-core-contracts v0.1.112
-	github.com/stretchr/testify v1.6.1
+	github.com/cloudevents/sdk-go/v2 v2.4.1
+	github.com/edgexfoundry/app-functions-sdk-go/v2 v2.0.0
+	github.com/edgexfoundry/go-mod-core-contracts/v2 v2.0.0
+	github.com/google/go-cmp v0.5.3 // indirect
+	github.com/google/uuid v1.2.0
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/json-iterator/go v1.1.11 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/stretchr/testify v1.7.0
+	go.uber.org/atomic v1.8.0 // indirect
+	go.uber.org/multierr v1.7.0 // indirect
+	go.uber.org/zap v1.18.1 // indirect
+	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b // indirect
```

Comment:
I like to clean up the file by removing all the `indirect` entries since they are not required.